nucypher/cli/commands/stake.py

```diff
@@ -102,13 +101,17 @@
     EXISTING_READABLE_FILE,
     WEI
 )
+from nucypher.cli.utils import setup_emitter
 from nucypher.config.characters import StakeHolderConfiguration
 
 option_value = click.option('--value', help="Token value of stake", type=click.INT)
 option_lock_periods = click.option('--lock-periods', help="Duration of stake in periods.", type=click.INT)
 option_worker_address = click.option('--worker-address', help="Address to bond as an Ursula-Worker", type=EIP55_CHECKSUM_ADDRESS)
 option_index = click.option('--index', help="The staker-specific stake index to edit", type=click.INT)
-option_only_lock = click.option('--only-lock', help="Locking tokens without deposit", is_flag=True)
+option_from_uncollected_rewards = click.option('--from-uncollected-rewards',
+                                               help="Only use uncollected staking rewards, and not tokens from staker address",
```

Comment:
I guess this is fine, but if to be precisely - unlocked tokens could be not only reward but also unlocked sub-stake

Reply:
Yes, this is truly just "unlocked and escrowed" NU.  It's hard to describe both clearly and tersely at the same time... :thinking:

Reply:
Something like `--from-alternative-source` (or if you like a slightly more descriptive name, `--from-unlocked-or-rewards`) and then use the help text to clarify? I think it'll be hard to cover the full picture in the option name.

Reply:
Used `from-unlocked` - ✔️